devicetypes/smartthings/zigbee-siren-src/zigbee-siren.groovy

```diff
@@ -0,0 +1,150 @@
+/**
+ *  Zigbee Siren
+ *
+ *  Copyright 2018 Samsung SRBR
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
+import physicalgraph.zigbee.zcl.DataType
+
+metadata {
+	definition(name: "ZigBee Siren", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "x.com.st.d.siren") {
+		capability "Actuator"
+		capability "Alarm"
+		capability "Configuration"
+		capability "Health Check"
+		capability "Refresh"
+
+		fingerprint profileId: "0104", inClusters: "0000,0003,0500,0502", outClusters: "0000", manufacturer: "ClimaxTechnology", model: "SRAC_00.00.00.16TC", deviceJoinName: "Ozom Smart Siren" // Ozom Siren - SRAC-23ZBS
+	}
+
+	tiles {
+		standardTile("alarm", "device.alarm", width: 2, height: 2) {
+			state "off", label:'off', action:'alarm.siren', icon:"st.secondary.siren", backgroundColor:"#ffffff"
+			state "siren", label:'siren!', action:'alarm.off', icon:"st.secondary.siren", backgroundColor:"#e86d13"
+		}
+
+		standardTile("refresh", "device.refresh", inactiveLabel: false, decoration: "flat") {
+			state "default", label: '', action: "refresh.refresh", icon: "st.secondary.refresh"
+		}
+
+		main "alarm"
+		details(["alarm","refresh"])
+	}
+}
+
+def installed() {
+	response(refresh())
+}
+
+def parse(String description) {
+	log.debug "parse ${description}"
+
+	def map = zigbee.getEvent(description)
+
+	if(!map) {
+		if(isZoneMessage(description)) {
+			map = parseIasMessage(description)
+		} else {
+			map = parseAttrMessage(description)
+		}
+	} 
+
+	def result = map ? createEvent(map) : [:]
+
+	if (description?.startsWith('enroll request')) {
+		def cmds = zigbee.enrollResponse()
+		log.debug "enroll response: ${cmds}"
+		result = cmds?.collect { new physicalgraph.device.HubAction(it)}
+	}
+
+	return result
+}
+
+private boolean isZoneMessage(description) {
+	return (description?.startsWith('zone status') || description?.startsWith('zone report'))
+}
+
+private Map parseIasMessage(String description) {
+	ZoneStatus zs = zigbee.parseZoneStatus(description)
+
+	def result = [:]
+    def value = ""
+	if(zs.isAlarm1Set() || zs.isAlarm2Set()) {    	
+		value = "siren"
+	} else {
+    	value = "off"
```

Comment:
spaces instead of tabs.

You'll probably just want to use a tool in your editor to convert spaces to tabs.